src/openrct2/peep/Staff.cpp

```diff
@@ -373,6 +373,13 @@ void staff_reset_stats()
     }
 }
 
+static std::pair<int32_t, int32_t> getPatrolAreaOffsetIndex(const CoordsXY& coords)
+{
+    // Patrol areas are 4 * 4 tiles (32 * 4) = 128 = 2^^7
+    auto hash = ((coords.x & 0x1F80) >> 7) | ((coords.y & 0x1F80) >> 1);
```

Comment:
It's not a hash.

Reply:
I called it hash as it's an amalgamation of the two coords smashed into one variable. It's not really a coord or anything else at this point.

Reply:
Hmm maybe something like `coordsCombined` then?

Reply:
Hm, fair enough. I don't think the name is ideal, but I get where it's coming from.